src/extension/kernel.ts

```diff
@@ -323,17 +324,22 @@ export class Kernel implements Disposable {
     if(this.#experiments.get('grpcRunner') && runner) {
       this.runner = runner
 
-      this.runnerReadyListener = runner.onReady(() => {
+      this.runnerReadyListener = runner.onReady(async () => {
         this.environment = undefined
 
-        runner.createEnvironment(
-          // copy env from process naively for now
-          // later we might want a more sophisticated approach/to bring this serverside
-          Object.entries(process.env).map(([k, v]) => `${k}=${v || ''}`)
-        ).then(env => {
+        try {
+          const env = await runner.createEnvironment(
+            // copy env from process naively for now
+            // later we might want a more sophisticated approach/to bring this serverside
+            Object.entries(process.env).map(([k, v]) => `${k}=${v || ''}`)
+          )
+
           if(this.runner !== runner) { return }
           this.environment = env
-        })
+        } catch(e: any) {
+          window.showErrorMessage(`[Runme] Failed to create environment for gRPC Runner: ${e.message}`)
+          console.error('Failed to create gRPC Runner environment', e)
```

Comment:
I've seen `[Runme] ` prefixes mostly in our internal logs, no strong preference though, feel free to disregard.

```suggestion
          window.showErrorMessage(`Failed to create environment for gRPC Runner: ${e.message}`)
          console.error('[Runme] Failed to create gRPC Runner environment', e)
```

Reply:
<img width="254" alt="Screenshot 2023-03-14 at 14 06 55" src="https://user-images.githubusercontent.com/731337/225136697-d373e78e-650e-4513-9610-ef764ed7e3df.png">